src/main/resources/application-dev.properties

```diff
@@ -11,3 +11,7 @@ IS_PRODUCTION=false
 
 app.uploaded.user.resources.directory.root=uploaded-resources-dir
 googlemaps.api.key=your_google_maps_api_key
+
+WORDS_API_KEY=your-words-api
+
+SWAGGER_DEV_SERVER=your-swagger-dev-server
```

Comment:
I think this will always be localhost, yeah? And honestly, we probably shouldn't even include it in the Prod environment, but we can work on that later.

Reply:
I updated them all with the local host. I thought I remembered you saying to keep them all consistent, but I can remove whatever we need too. Would removing local host break anything now?